application/client/src/app/service/session/dependencies/search/filters/request.ts

```diff
@@ -77,16 +77,41 @@ export class FilterRequest
     private _hash!: string;
     public found: number = 0;
 
-    static isValid(request: string | undefined): boolean {
-        if (request === undefined) {
-            return false;
-        }
-        if (!regex.isValid(request)) {
+    static isValid(
```

Comment:
Can it get as an argument just `IFilter`? Almost everywhere you are passing `IFilter` into it